project.clj

```diff
@@ -1,39 +1,21 @@
 (defproject pubsub-test "__VERSION__"
-  :description "meters-workers for entity-store"
-  :url "http://example.com/FIXME"
+  :description "pubsub harness for reducing duplicate messages"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.8.0"]
                  [cheshire "5.7.0"]
                  [medley "0.8.4"]
-                 [com.google.cloud/google-cloud "0.30.0-alpha"]
-                 [com.google.guava/guava "23.4-jre"]]
+                 [com.google.cloud/google-cloud-pubsub "0.32.0-beta"]
+                 ;; logging
+                 [com.taoensso/timbre "4.8.0"]
+                 [com.fzakaria/slf4j-timbre "0.3.4"]
+                 [log4j/log4j "1.2.17"]
+                 [org.slf4j/jul-to-slf4j "1.7.25"]
+                 [org.slf4j/jcl-over-slf4j "1.7.25"]
```

Comment:
It's possible that it's the (combination) of libraries we're using that were the problem, might be worth considering this when you test to see what has fixed it

Reply:
I don't think so as the java one only has the basic dependencies and I was still able to create the dup issue.